src/app/component/MapView/MapView.js

```diff
@@ -3,10 +3,11 @@ import { Map, TileLayer, Marker, Popup } from "react-leaflet";
 import { MapStyle } from "./MapView.style";
 // import Routing from './MapView-routing'
 
-function MapView() {
+function MapView(props) {
+  console.log(props)
```

Comment:
remove this

Reply:
Strange I do not have this on my end, once i make another commit it will be gone.